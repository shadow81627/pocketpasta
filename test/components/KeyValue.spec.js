import KeyValue from '@/components/KeyValue.vue';
import { shallowMount } from '@vue/test-utils';

const factory = () =>
  shallowMount(KeyValue, {
    propsData: {
      label: 'Job Title',
      value: 'Full Stack Web Developer',
    },
  });

describe('KeyValue', () => {
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
