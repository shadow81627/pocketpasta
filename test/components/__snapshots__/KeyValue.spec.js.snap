// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KeyValue renders properly 1`] = `"<p><strong>Job Title:<span>&nbsp;</span></strong> <span>Full Stack Web Developer</span></p>"`;
